java/src/org/openqa/selenium/docker/ContainerConfig.java

```diff
@@ -59,6 +81,7 @@ public ContainerConfig(
     this.networkName = networkName;
     this.autoRemove = true;
     this.shmSize = shmSize;
+    this.hostConfig = hostConfig != null ? hostConfig : ImmutableMap.of();
```

Comment:
Is there a chance `hostConfig` is null? Why don't we do a `Require.nonNull`?

Reply:
Yes, there is no chance `hostConfig` is null
Require.nonNull is also checked in ContainerConfig `this.hostConfig = Require.nonNull("Host config", hostConfig);`